app/start.py

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import urlparse
 import const
+import urlparse
```

Comment:
Looks like urlparse and utils are no longer used. Can you remove them?

Reply:
done.